toolchest_client/tools/tool.py

```diff
@@ -35,13 +35,13 @@ def __init__(self, tool_name, tool_version, tool_args, output_name,
                  max_input_bytes_per_file=FOUR_POINT_FIVE_GIGABYTES,
                  max_input_bytes_per_file_parallel=FOUR_POINT_FIVE_GIGABYTES,
                  group_paired_ends=False, compress_inputs=False,
-                 output_type=OutputType.FLAT_TEXT, output_is_directory=False,
+                 output_type=OutputType.FLAT_TEXT, output_is_directory=True,
                  output_names=None):
         self.tool_name = tool_name
         self.tool_version = tool_version
         self.tool_args = tool_args
         self.output_name = output_name
-        self.output_path = output_path
+        self.output_path = os.path.abspath(output_path)
```

Comment:
@bcai2 fyi this broke not specifying an output path. fixing in the async PR + adding an integration test to catch this case in the future